airdrop-claim-reject-cancel-hts-token-js-sdk/package.json

```diff
@@ -0,0 +1,17 @@
+{
+	"name": "hedera-example-hts-airdrops-hip-904",
+	"version": "1.0.0",
+	"description": "",
+	"main": "airdrop.js",
+	"type": "module",
+	"scripts": {
```

Comment:
Probably can remove this scripts if it's not being used

Reply:
Better yet: add an npm run script which runs the main script.

Reply:
Added `"start": "node airdrop.js"`